src/OrbitDB.js

```diff
@@ -226,11 +238,19 @@ class OrbitDB {
       onDrop: this._onDrop.bind(this),
       onLoad: this._onLoad.bind(this)
     })
+
+    const relayEvents = options.relayEvents ? options.relayEvents : this._relayEvents
+
     const identity = options.identity || this.identity
 
     const store = new Store(this._ipfs, identity, address, opts)
     store.events.on('write', this._onWrite.bind(this))
 
+    const eventRelay = this.events
```

Comment:
what is the purpose of declaring `eventRelay`

Reply:
so I don't have to use `this` in the arrow function below it, `this` goes out of in the closure, so a separate reference to `this.events` is required.

Reply:
what would you say `this` is referring to when inside of the for loop statements brackets?

Reply:
Inside the for-loop it refers to the orbitdb instance.
However, inside the closure inside the for loop, the arrow function, it refers to the context of the arrow function

Reply:
arrow functions have lexical scope so you should be good here

Reply:
My bad, arrow functions don't change scope like ordinary functions do.
example: 
```
const { EventEmitter } = require('events')
(function () {
  const address = 'zdpu.....ect'
  this.events = new EventEmitter()
  const store = { events: new EventEmitter() }
  const relayEvents = ['ready']

  for (const eventType of relayEvents) {
    store.events.on(eventType, function (...args) {
      console.log(typeof this.events) //undefined
      this.events.emit(eventType, address, ...args) //TypeError: Cannot read property 'emit' of undefined
    })
  }

  this.events.on('ready', (...args) => console.log(`Got ready event with args ${args}`))

  store.events.emit('ready', 'this is a test')
}) ()
```

Reply:
So it turns out that arrow functions don't change the scope of this like standard functions do.
e.g.
```
const { EventEmitter } = require('events')

function abc () {
  const address = 'zdpu.....ect'
  this.events = new EventEmitter()
  const store = { events: new EventEmitter() }
  const relayEvents = ['ready']

  for (const eventType of relayEvents) {
    store.events.on(eventType, function (...args) {
      console.log(typeof this.events)                       //undefined
      this.events.emit(eventType, address, ...args)} //TypeError: Cannot read property 'emit' of undefined
    )
  }

  this.events.on('ready',(...args) => console.log(`Got ready event with args ${args}`))

  store.events.emit('ready', 'this is a test')
}
abc()
``` 

e.g. 2 works fine
```
const { EventEmitter } = require('events')

function abc () {
  const address = 'zdpu.....ect'
  this.events = new EventEmitter()
  const store = { events: new EventEmitter() }
  const relayEvents = ['ready']

  for (const eventType of relayEvents) {
    store.events.on(eventType, (...args) => {
      console.log(typeof this.events)                          //object
      this.events.emit(eventType, address, ...args)}
    )
  }

  this.events.on('ready',(...args) => console.log(`Got ready event with args ${args}`))

  store.events.emit('ready', 'this is a test')
}
abc()
```

Reply:
Fixed!